src/WidenImage.js

```diff
@@ -0,0 +1,86 @@
+import './WidenImage.css'
+import utilities from './Utilities'
+
+// If enabled, widened images will have thin red dashed border
+const enableDebugBorders = false
+
+const widenAncestors = (el) => {
+  while ((el = el.parentElement) && !el.classList.contains('content_block')) {
+    // Only widen if there was a width setting. Keeps changes minimal.
+    if (el.style.width) {
+      el.style.width = '100%'
+    }
+    if (el.style.maxWidth) {
+      el.style.maxWidth = '100%'
+    }
+    if (el.style.float) {
+      el.style.float = 'none'
+    }
+  }
+}
+
+const shouldWidenImage = (image) => {
+  // Some images are within a <div class='noresize'>...</div> which indicates
+  // they should not be widened. Example below has links overlaying such an image.
+  // See:
+  //   'enwiki > Counties of England > Scope and structure > Local government'
+  if (utilities.findClosest(image, "[class*='noresize']")) {
```

Comment:
+1 to renaming `utilities` to `elementUtilities`. Narrow scope FTW!

iirc with `findClosest` I was trying to imitate [JQuery](https://www.w3schools.com/jquery/traversing_closest.asp) and [Mozilla](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) which just use `closest`.

But point taken on names giving context... `closestAncestorMatchingSelector` is a bit verbose but pretty clear. Do you like that less/more than `queryClosestSelector`?